config/config.go

```diff
@@ -0,0 +1,46 @@
+package config
```

Comment:
Lets move the config stuff into utils for now. Config as a package isn't going to grow and i don't feel like it justifies having its own package for this.